### PR TITLE
Add commit message guidelines to contributing

### PR DESCRIPTION
**Pull request by a-mere-peasant**

### CODE REVIEW

**Comment by pcahyna** [comment]
Maybe this belongs to https://github.com/linux-system-roles/linux-system-roles.github.io - @tyll @richm what do you think?

**Comment by richm** [comment]
> Maybe this belongs to https://github.com/linux-system-roles/linux-system-roles.github.io - @tyll @richm what do you think?

Yes, if we don't already have git commit message guidelines

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    vary here
+
+ If you use an issue tracker, put references to them at the bottom,
+ like this:
+
+ Resolves: #123
+ See also: #456, #789
+
+```
+This is taken from [chris beams git commit](https://chris.beams.io/posts/git-commit/) You may want to read this for a more deatiled explanation (and links to other posts on how to write a good commit message).
```
Comment (tyll) [inline contributing.md:43]: Please wrap this line at 88 characters. Also please add a fullstop before `You` and there is a type in `detailed`.

**Comment by tyll** [comment]
> Maybe this belongs to https://github.com/linux-system-roles/linux-system-roles.github.io - @tyll @richm what do you think?

Sure, if all the other roles/projects want to follow this, too. Still this role would need a pointer to the generic document when it is created.

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ If you use an issue tracker, put references to them at the bottom,
+ like this:
+
+ Resolves: #123
+ See also: #456, #789
+
+```
+This is taken from [chris beams git commit](https://chris.beams.io/posts/git-commit/).
+You may want to read this for a more detailed explanation (and links to other posts on
+how to write a good commit message).
```
Comment (tyll) [inline contributing.md:45]: Please add something like

`This content is licensed under [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/).`

See https://github.com/cbeams/chris.beams.io/issues/48

Comment (a-mere-peasant) [reply contributing.md:45]: I have added the license information .

**Comment by a-mere-peasant** [comment]
> > Maybe this belongs to https://github.com/linux-system-roles/linux-system-roles.github.io - @tyll @richm what do you think?
> 
> Sure, if all the other roles/projects want to follow this, too. Still this role would need a pointer to the generic document when it is created.

Maybe we can keep the irc and mailing list info at both places in the readme and  the rest can be kept at https://github.com/linux-system-roles/linux-system-roles.github.io ?
If so should I remove this commit from here ? 
and would a new page specific to the contributing guidelines have to be made because the content according to #192 would be too much to use in the main page ?

File: contributing.md (hunk 2 above)
Comment (tyll) [reply contributing.md:45]: thank you

**Comment by tyll** [comment]
@a-mere-peasant the change looks good. Please rebase on the latest master branch. Also, please add your name to the git config and update the commit as described in https://www.git-tower.com/learn/git/faq/change-author-name-email

**Comment by tyll** [comment]
@a-mere-peasant thx, are you going to update the name information on the commit, too?

**Comment by a-mere-peasant** [comment]
> @a-mere-peasant thx, are you going to update the name information on the commit, too?

sure, sorry for not changing it earlier.

**Comment by tyll** [comment]
> > @a-mere-peasant thx, are you going to update the name information on the commit, too?
> 
> sure, sorry for not changing it earlier.

ok, please tell me when you did it. Currently the commit uses "a-mere-peasant" as name for the author.

**Comment by a-mere-peasant** [comment]
> > > @a-mere-peasant thx, are you going to update the name information on the commit, too?
> > 
> > 
> > sure, sorry for not changing it earlier.
> 
> ok, please tell me when you did it. Currently the commit uses "a-mere-peasant" as name for the author.

I thought using my username was fine since I use that wherever I contribute ,I'll change it right away .Sorry for the trouble

**Pull request merged**